draft-ietf-jsonpath-base.md

```diff
@@ -1474,23 +1474,29 @@ Queries:
 
 ## Normalized Paths
 
-A Normalized Path is a JSONPath with restricted syntax that identifies a node by providing a query that results in exactly that node. For example,
-the JSONPath expression `$.book[?(@.price<10)]` could select two values with Normalized Paths
-`$['book'][3]` and `$['book'][5]`. For a given JSON value, there is a one to one correspondence between the value's
-nodes and the Normalized Paths that identify these nodes.
-Note that there is precisely one Normalized Path that identifies each node.
+A Normalized Path is a conventional representation of the identity of a node in a value.
+Specifically, a Normalized Path is a JSONPath query with restricted syntax (defined below),
+e.g., `$['book'][3]`, which when applied to the value results in a nodelist consisting
+of just the node identified by the Normalized Path.
+Note that a Normalized Path represents the identity of a node _in a specific value_.
 
-A JSONPath implementation may output Normalized Paths instead of, or in addition to, the values identified by these paths.
+Normalized Paths provide a predictable format that simplifies testing and post-processing
```

Comment:
This is a sentence about usage; at this point it isn't quite clear that the normalized path of a node is deterministic, i.e., for the same node in the same value, two implementations will generate the exact same normalized path.

Reply:
Added a sentence at the end of the first paragraph of this section to make that clear.